src/main/java/uk/gov/companieshouse/web/pps/controller/pps/ConfirmationController.java

```diff
@@ -113,17 +126,15 @@ public String getConfirmation(@PathVariable String companyNumber,
 
     private String setUpPaymentDateDisplay(PayableLateFilingPenalty payableLateFilingPenalty) {
         if (payableLateFilingPenalty.getPayment() != null) {
-            return StringUtils.isEmpty(payableLateFilingPenalty.getPayment().getPaidAt()) ? "" :
-                    OffsetDateTime.parse(payableLateFilingPenalty.getPayment().getPaidAt(), DateTimeFormatter.ISO_OFFSET_DATE_TIME)
-                            .format(DateTimeFormatter.ofPattern("d MMMM uuuu", Locale.UK));
+            return LocalDate.now()
+                    .format(DateTimeFormatter.ofPattern("d MMMM uuuu", Locale.UK));
         }
         return "";
     }
 
-    private String setUpPaymentAmountDisplay(PayableLateFilingPenalty payableLateFilingPenalty) {
-        if (payableLateFilingPenalty.getPayment() != null) {
-            return StringUtils.isEmpty(payableLateFilingPenalty.getPayment().getAmount()) ? "" :
-                    "£" + payableLateFilingPenalty.getPayment().getAmount() + " (no VAT is charged)";
+    private String setUpPaymentAmountDisplay(LateFilingPenalty payableLateFilingPenalty) {
```

Comment:
Could this method be pulled out into the 'PenaltyUtils'? For me there is too much logic in the controller. I am hoping that we create a service layer for all this post the first release.